mirrord/agent/src/steal.rs

```diff
@@ -242,30 +250,37 @@ impl StealWorker {
                 }
             }
         }
-        debug!("TCP Stealer exiting");
+
         Ok(())
     }
 
+    #[tracing::instrument(level = "debug", skip(self))]
     pub async fn handle_client_message(&mut self, message: LayerTcpSteal) -> Result<()> {
         use LayerTcpSteal::*;
         match message {
-            PortSubscribe(port) => {
+            PortSubscribe((port, traffic_filter)) => {
```

Comment:
I'm thinking if it needs to be two messages or whole new one as we'll need to update it "on the fly" in operator mode.
Okay I think it's fine for now - if port already subscribed it will just update the filter.

Reply:
(no need to add it, just thinking out loud for future, maybe add a comment that that's how it will work)